chain/client/src/metrics.rs

```diff
@@ -592,6 +592,18 @@ pub(crate) static ORPHAN_CHUNK_STATE_WITNESSES_TOTAL_COUNT: Lazy<IntCounterVec>
     .unwrap()
 });
 
+pub(crate) static CHUNK_STATE_WITNESS_NETWORK_ROUNDTRIP_TIME: Lazy<HistogramVec> =
+    Lazy::new(|| {
+        try_create_histogram_vec(
+            "near_chunk_state_witness_network_roundtrip_time",
+            "Time in seconds between sending state witness through the network to chunk producer \
```

Comment:
is second small enough to make meaningful measurement? should we consider millisec?

Reply:
float seconds should handle the fractions, but will change to milliseconds to be more explicit.

Reply:
sure. and this may be completely nit, but if we expect the value to be in decimals most of the time, I wonder what we get by having float sec instead of int millisec (or float millisec)

Reply:
Yeah you are right. I was actually trying to be consistent with other places tracking the latency, but now changed to exact milliseconds.

Reply:
https://prometheus.io/docs/practices/naming/ FWIW is a good document on the naming and units in prometheus metrics. It explicitly suggests using base SI units (and I generally agree -- its easier to work with the metrics and visualization when you don't need to double guess whether the value is in millis or base units.)

Reply:
+1 to what nagisa said, the common practice in our dashboards is to use seconds
You can configure the grafana panel to be aware that the units are seconds and it will nicely display values in miliseconds if they are sub second. e.g. "time to apply chunks per shard" in the [blockhain utilization dashboard](https://nearinc.grafana.net/goto/dQ5e701Sg?orgId=1)

Reply:
ok, moved back to seconds as it looks like the common practice. thanks for the links and suggestions.